sound/soc/sof/intel/hda.c

```diff
@@ -257,6 +292,10 @@ int hda_sdw_startup(struct snd_sof_dev *sdev)
 	if (pdata->machine && !pdata->machine->mach_params.link_mask)
 		return 0;
 
+	ret = hda_sdw_read_lcount(sdev);
```

Comment:
Where will the `ret` be used? And will it be better to rename `ret` to `lcount`?

Reply:
> Where will the `ret` be used? And will it be better to rename `ret` to `lcount`?

Oh, I see, we only check it. Not really using it.

Reply:
I should rename this 'check_sdw_lcount', we've not using the result just verifying that what ACPI reported is aligned with what the hardware supports. Good suggestion.